.appveyor.yml

```diff
@@ -17,7 +14,6 @@ build:
   parallel: true
 
 build_script:
-  - cd c:\project\pcl
```

Comment:
BTW this should be `c:\projects\pcl` (plural)

Reply:
Thanks. It never got there though :cry: